src/routes/Trip/Items.tsx

```diff
@@ -49,6 +51,14 @@ const Items = ({
 
   const isListPresent = currentTrip && listsById[listId];
 
+  let listOfItemIds = [];
```

Comment:
we can remove this declaration here and leave it as a `const` below when you do the assigning.